tests/Avalonia.Visuals.UnitTests/VectorTests.cs

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) The Avalonia Project. All rights reserved.
+// Licensed under the MIT license. See licence.md file in the project root for full license information.
+
+using Xunit;
+using Avalonia;
+using System;
+
+namespace Avalonia.Visuals.UnitTests
```

Comment:
`Vector` is located in the 'Avalonia' root namespace. The type itself is in the 'Avalonia.Visuals' Project. Is the namespace for the Tests Ok ?

Reply:
The namespace looks okay